File: gen_wallet.py
from algosdk import account, mnemonic

private_key, address = account.generate_account()
mnemonic_phrase = mnemonic.from_private_key(private_key)

print("=" * 60)
print("   PayPerAPI Algorand Testnet Wallet")
print("=" * 60)
print(f"\n  Address  : {address}")
print(f"\n  Mnemonic : {mnemonic_phrase}")
print("\n" + "=" * 60)
print("  SAVE THESE - never share your mnemonic!")
print("=" * 60)

File: backend/gateway/gen-wallet.mjs
import algosdk from 'algosdk';

const account = algosdk.generateAccount();
const mnemonic = algosdk.secretKeyToMnemonic(account.sk);
const address = account.addr.toString();

console.log('\n=== NEW ALGORAND TESTNET CLIENT WALLET ===');
console.log('Address  :', address);
console.log('Mnemonic :', mnemonic);
console.log('\nFund this wallet at: https://bank.testnet.algorand.network/');
console.log('Paste address above and click "Dispense"\n');
